src/healthbot/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure CLI healthbot Extension #
+This is the extension for healthbot
```

Comment:
I suggest write more words. What is healthbot? What can it do? Who need it? Why we need it? What is typical use case?